queries.sql

```diff
@@ -0,0 +1,68 @@
+BEGIN;
+
+UPDATE animals
+SET species = 'unspecified';
+
```

Comment:
- [ ] It would be better if we verify the change was made before rolling back the transaction. Here is how we can achieve that for example; `SELECT species from animals`, here we are verifying if all the species are set to 'unspecified' before rolling it back